salt/beacons/avahi_announce.py

```diff
@@ -16,6 +16,7 @@
 import logging
 import time
 
+import salt.utils
```

Comment:
We should be importing `salt.utils.stringutils` here instead of `salt.utils`:

```
>>> import salt.utils
>>> salt.utils.stringutils.to_unicode
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'stringutils'
```